actors/verifreg/src/lib.rs

```diff
@@ -672,13 +672,51 @@ impl Actor {
                 term_min: req.term_min,
                 term_max: req.term_max,
                 expiration: req.expiration,
-            })
+            });
+            allocation_total += DataCap::from(req.size.0);
+        }
+
+        let st: State = rt.state()?;
+        let mut claims = st.load_claims(rt.store())?;
+        let mut updated_claims = Vec::<(ClaimID, Claim)>::new();
+        for req in &reqs.extensions {
+            let provider_id = rt
```

Comment:
Double checking: we don't check client here but that's ok because it is part of the FIP that this operation can be done by any address with datacap?

Reply:
Yes, I added a comment.